src/attribute_type.rs

```diff
@@ -4410,4 +4410,36 @@ mod tests {
     fn test_parse_unknown_attribute() {
         assert!(parse_attribute("unknown").is_err())
     }
+
+    #[test]
+    fn test_serialize_fmtp_parameters_unknown_tokens() {
+        let att = SdpAttributeFmtpParameters {
+            packetization_mode: 1,
+            level_asymmetry_allowed: false,
+            profile_level_id: 0x0042_0010,
+            max_fs: 0,
+            max_cpb: 0,
+            max_dpb: 0,
+            max_br: 0,
+            max_mbps: 0,
+            usedtx: false,
+            stereo: false,
+            useinbandfec: false,
+            cbr: false,
+            max_fr: 0,
+            maxplaybackrate: 48000,
+            maxaveragebitrate: 0,
+            ptime: 0,
+            minptime: 0,
+            maxptime: 0,
+            encodings: Vec::new(),
+            dtmf_tones: "".to_string(),
+            rtx: None,
+            unknown_tokens: vec!["sprop-parameter-sets=Z0LAFYyNQKD5APCIRqA=,aM48gA==".to_string()],
+        };
+        assert_eq!(
+            format!("{}", att),
+            "packetization-mode=1;sprop-parameter-sets=Z0LAFYyNQKD5APCIRqA=,aM48gA=="
+        );
+    }
```

Comment:
I would propose to make this simpler with something like this:

`check_parse_and_serialize("fmtp:102 packetization-mode=1;profile-level-id=42c015;sprop-parameter-sets=Z0LAFYyNQKD5APCIRqA=,aM48gA==");`

But I would like to see more of these including combinations with red and dtmf with other parameters to make sure we are not screwing up these.